action/protocol/poll/slasher.go

```diff
@@ -465,27 +465,24 @@ func (sh *Slasher) calculateUnproductiveDelegates(ctx context.Context, sr protoc
 			bcCtx.Tip.Height,
 			sh.productivity,
 		)
-		if err != nil {
-			return nil, err
-		}
-		// The current block is not included, so add it
-		numBlks++
-		if _, ok := produce[blkCtx.Producer.String()]; ok {
-			produce[blkCtx.Producer.String()]++
-		} else {
-			produce[blkCtx.Producer.String()] = 1
-		}
 	} else {
 		numBlks, produce, err = rp.ProductivityByEpoch(
 			epochNum,
-			bcCtx.Tip.Height+1,
+			bcCtx.Tip.Height,
```

Comment:
i saw v1.0.0 used Tip.Height, and change to +1 in https://github.com/iotexproject/iotex-core/commit/8bc77d3e24360bd8959eee49bae7ec2efc9975f9 20 days again

is it possible to add test case to cover this?